js/plugin.iiif-iabookreader_strawberry.js

```diff
@@ -288,27 +330,80 @@ BookReader.prototype.loadManifest = function () {
 BookReader.prototype.setupTooltips = function() {
 };
 
+function processBehaviorToIABookreaderModeV3(jsonLd) {
+  // Bookreader supports only two modes but also a book cover or not
+  // IIIF Presentation API V2 and V3 supports more
+  // So we need to reduce/map back to only 2 options
+  // V3 key at manifest level/and/or Sequence level is `behavior`
+  // V2 key at manifest level/and/or Sequence is `viewingHint`
+  if (typeof (jsonLd.behavior) !== "undefined") {
+    if (typeof jsonLd.behavior === 'string') {
+      if (jsonLd.behavior == 'paged') {
+        return 2;
+      }
+      if (jsonLd.behavior == 'individuals') {
+        return 1;
+      }
+    } else if (jsonLd.behavior.constructor.name == "Array") {
+      if (jsonLd.behavior.includes("paged")) {
+        return 2;
+      }
+      if (jsonLd.behavior.includes("individuals")) {
+        return 1;
+      }
+    }
+  }
+
+return 2;
+}
+
+function processBehaviorToIABookreaderModeV2(jsonLd) {
```

Comment:
Tested with 
```
"viewingHint": "paged",
```
and 
```
"viewingHint": "individuals",
```
in a IIIF v2.1 manifest metadatadisplay template and I can confirm this is working.